app/Nova/Travel.php

```diff
@@ -228,6 +228,87 @@ public function fields(Request $request): array
                 ]
             ),
 
+            new Panel(
+                'International Travel',
+                [
+                    Boolean::make('Destination Outside United States', 'is_international')
+                        ->help(
+                            'Check this box if your destination is outside of the United States.'
+                        )
+                        ->hideFromIndex(),
+
+                    Text::make('Justification', 'international_travel_justification')
+                        ->required()
+                        ->rules('required_if:is_international,1', 'nullable')
+                        ->help(
+                            'Please explain how this travel meets essential travel criteria as defined by Georgia Tech.'
+                        )
+                        ->hideFromIndex(),
+
+                    Boolean::make('Export-Controlled Technology', 'export_controlled_technology')
+                        ->required()
+                        ->help(
+                            'Do you plan to take any information or technology that is controlled?'
+                        )
+                        ->hideFromIndex(),
+
+                    Text::make('Export-Controlled Technology Description', 'export_controlled_technology_description')
+                        ->required()
+                        ->rules('required_if:export_controlled_technology,1', 'nullable')
+                        ->help(
+                            'If yes, please describe the information or technology.'
+                        )
+                        ->hideFromIndex(),
+
+                    Boolean::make('Embargoed Destination', 'embargoed_destination')
+                        ->required()
+                        ->help(
+                            'Do you plan to travel to an embargoed destination?'
```

Comment:
Is the assumption that the person completing this knows whether or not where they're going is an embargoed destination? Is that even allowed?

Reply:
Same as above. Not sure what you mean by the latter question.

Reply:
Are we/GT allowed to travel to embargoed destinations?

Reply:
If you get a license from OFAC, sure.